scripts/lib/api/processHtml.test.ts

```diff
@@ -191,6 +192,98 @@ test("removeColonSpans()", () => {
   doc.expectHtml(`<dt class="field-odd">Parameters</dt>`);
 });
 
+test("removeMatplotlibFigCaptions()", () => {
```

Comment:
It'd be helpful to use `describe()` to group the two tests together - you can find some examples in this file. Then, in the `describe()` block, define a variable for `inputHtml` with the multiline string. That will allow you to avoid repeating the same HTML three times.

Reply:
I've grouped the tests. I didn't define the variable for the inputHtml because the HTML is actually different every time. The second test edits it so that it doesn't match the selector. Still, even for that test the HTML gets modified. Specifically,
```html
<img alt="../_images/fake_provider-1_00.png" class="plot-directive" src="../_images/fake_provider-1_00.png" />
```
gets changed to
```html
<img alt="../_images/fake_provider-1_00.png" class="plot-directive" src="../_images/fake_provider-1_00.png">
```
(note the missing backslash at the end). I don't know why that happens.